lib/azure.rb

```diff
@@ -22,6 +22,7 @@
 require 'addressable/uri'
 require 'faraday'
 require 'faraday_middleware'
+require 'azure/core'
```

Comment:
@vishrutshah Per our offline conversations, I prefer not to remove 'require' lines in this change, so we don't introduce a hard dependency on a new release of azure-core (that would autoload those classes).